sdk/attestation/attestation/recordings/node/aad_attestation_client/recording_attestsgxaad.js

```diff
@@ -0,0 +1,58 @@
+let nock = require('nock');
+
+module.exports.hash = "b49c79a7a878d5f0442df325d9f65fa7";
+
+module.exports.testInfo = {"uniqueName":{},"newDate":{}}
+
+nock('https://login.microsoftonline.com:443', {"encodedQueryParams":true})
+  .post('/72f988bf-86f1-41af-91ab-2d7cd011db47/oauth2/v2.0/token', "response_type=token&grant_type=client_credentials&client_id=9069aeeb-dcde-4145-9667-0f1d095f926b&client_secret=39e74e6b-a8ca-4aa8-99f2-0c2faf38a3c5&scope=https%3A%2F%2Fattest.azure.net%2F.default")
```

Comment:
Looks like the `replaceableVariables` dictionary is missing these?

Reply:
Fixed.